docs/tutorial/part-one/index.md

```diff
@@ -206,17 +206,17 @@ export default () => (
 
 In the browser, the “About Gatsby” header text should now be replaced with “This is a header.” But we don’t want the “About” page to say “This is a header.” We want it to say, “About Gatsby”.
 
-4.  Head back to `/src/components/header.js`, and make the following change:
```

Comment:
I changed these from absolute paths to relative. Good idea or bad idea? If nothing else, we should be consistent throughout the docs and throughout the tutorial.